jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SendMessagePane.kt

```diff
@@ -68,28 +68,32 @@ class SendMessagePane(
     }
 
     suspend fun sendMessage() {
-        if (validateFields()) {
-            try {
-                withContext(Dispatchers.IO) {
-                    val messageId = client.sendMessage {
-                        it.queueUrl(queue.queueUrl)
-                        it.messageBody(inputText.text)
-                        if (queue.isFifo) {
-                            it.messageDeduplicationId(fifoFields.deduplicationId.text)
-                            it.messageGroupId(fifoFields.groupId.text)
-                        }
-                    }.messageId()
-                    messageSentLabel.text = message("sqs.send.message.success", messageId)
-                }
-                clear(isSend = true)
-            } catch (e: Exception) {
-                messageSentLabel.text = message("sqs.failed_to_send_message")
+        if (!validateFields()) {
+            return
+        }
+        try {
+            withContext(Dispatchers.IO) {
+                val messageId = client.sendMessage {
+                    it.queueUrl(queue.queueUrl)
+                    it.messageBody(inputText.text)
+                    if (queue.isFifo) {
+                        it.messageDeduplicationId(fifoFields.deduplicationId.text)
+                        it.messageGroupId(fifoFields.groupId.text)
+                    }
+                }.messageId()
+                messageSentLabel.text = message("sqs.send.message.success", messageId)
+            }
+            clear(isSend = true)
```

Comment:
When i was playing with it this afternoon - this caused the UI to freeze. I had to forcibly put clear on the UI thread.

It's concerning that this kind of issue can creep in in coroutine land; I don't remember seeing a full UI lock-up in bugs we've had before.

Reply:
That is very strange, I'll put that into the base PR of this PR